pkg/cmd/cli/cmd/expose.go

```diff
@@ -179,8 +179,8 @@ func validateFlags(cmd *cobra.Command, generator string) error {
 		if cmdutil.GetFlagInt(cmd, "port") != -1 {
 			invalidFlags = append(invalidFlags, "--port")
 		}
-		if cmdutil.GetFlagBool(cmd, "create-external-load-balancer") {
-			invalidFlags = append(invalidFlags, "--create-external-load-balancer")
+		if cmdutil.GetFlagBool(cmd, "type") {
```

Comment:
Do this in a separate pull.  I don't think this what we want to do.  If we have an old flag and a new flag with the same meaning, alias one with the other.  Don't clutter this pull with it though.